recipes/liblqr/all/conanfile.py

```diff
@@ -0,0 +1,126 @@
+from conan import ConanFile
+from conan.tools.apple import fix_apple_shared_install_name
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.env import VirtualBuildEnv
+from conan.tools.files import copy, get, rm, rmdir
+from conan.tools.gnu import Autotools, AutotoolsToolchain, PkgConfigDeps
+from conan.tools.layout import basic_layout
+import os
+
+required_conan_version = ">=1.54.0"
+
+
+class LibLqrConan(ConanFile):
+    name = "liblqr"
+    description = (
+        "The LiquidRescale (lqr) library provides a C/C++ API for performing "
+        "non-uniform resizing of images by the seam-carving technique."
+    )
+    license = ("LGPL-3.0", "GPL-3.0")
```

Comment:
```suggestion
    license = ("LGPL-3.0-or-later", "GPL-3.0-or-later")
```
Can be made a bit more specific.

>  This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.